action.yml

```diff
@@ -0,0 +1,60 @@
+name: "Update docs"
+description: "Generate markdown documentation, commit it to a branch of the gitbook repo, and push it"
+inputs:
+  gitbook-branch:
+    description: "Which branch of wandb/gitbook to update"
+    required: true
+    default: "en"
+  access-token:
+    description: "Personal access token to use when checking out repos"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: checkout gitbook repo
+      uses: actions/checkout@v2
+      with:
+        repository: wandb/gitbook
+        path: repos/gitbook
+        ref: ${{ inputs.gitbook-branch }}
+        token: ${{ inputs.access-token }}
+    # setup: bring in python plus the requirements for generating docs and the new release
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.9"
+    - name: setup python environment, including wandb client
+      shell: bash
+      env:
+        VERSION: latest
+      run: python -m pip install -r ${{ github.action_path }}/requirements.txt git+https://github.com/wandb/client.git@$VERSION
+
+    # generate the docs from the latest client library and overwrite gitbook contents
+    - name: generate documentation
+      shell: bash
+      env:
+        VERSION: latest
+        DOCUGEN_CONFIG_PATH: ${{ github.action_path }}/config.ini
+      run: python ${{ github.action_path }}/generate.py --template_file ./repos/gitbook/SUMMARY.md --commit_id $VERSION --output_dir ./repos/gitbook
+    # for debugging/tracking, display the generated table of contents
+    - name: display ToC
+      shell: bash
+      run: cat ./repos/gitbook/SUMMARY.md
+
+    # stage: commit the changes
+    - name: commit changes
+      shell: bash
+      working-directory: ./repos/gitbook
+      run: |
+        git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Is this from [here](https://api.github.com/users/better-informatics%5Bbot%5D)? Just looks a bit weirds :)